docs/RELEASE.md

```diff
@@ -1,3 +1,50 @@
+# Development Road Map
+The following is a summary of the NNI team's backlog for the next 6 months. Some completed items (either in a release or master branch) are included to provide the context and progress of the work.
+## New Tuners Supports
+* New MSRA research Algo
+* Hyperband
+* Grid search
+## New Training Services Supports
+* Kubeflow
+## User Interface and Experiences
+* Design nni APIs for users to specify search space in their code
+* Web UX refactor
+* V0.4.0 (November 2018)
+
+# Release 0.3.0 - 11/2/2018
+## Major Features
+* Support running multiple experiments simultaneously. You can run multiple experiments by specifying a unique port for each experiment:
+
+    ```nnictl create --port 8081 --config <config file path>```
+
+    You can still run the first experiment without '--port' parameter:
+
+    ```nnictl create --config <config file path>```
+* A builtin Batch Tuner which iterates all parameter combination, can be used to submit batch trial jobs.
+* nni.report_final_result(result) API supports more data types for result parameter, it can be of following types:
+    * int
+    * float
+    * A python dict containing 'default' key, the value of 'default' key should be of type int or float. The dict can contain any other key value pairs.
+* Continuous Integration
+    * Switched to Azure pipelines
+* Others
+    * <span style="color:red">API nn.get_parameters() is renamed to nni.get_next_parameter(), this is a broken change, all examples of prior releases can not run on v0.3, please clone nni repo to get new examples.</span>
```

Comment:
We need to add this in a section called "Breaking Changes".

Reply:
Thanks, Done